docs/environment-variables.md

```diff
@@ -22,6 +22,7 @@ most users. Environment variables may be removed at any time.
 | `ARGO_AGENT_PATCH_RATE`                | `time.Duration`     | `DEFAULT_REQUEUE_TIME`                                                                      | Rate that the Argo Agent will patch the workflow task-set.                                                                                                                                                                                                               |
 | `ARGO_AGENT_CPU_LIMIT`                 | `resource.Quantity` | `100m`                                                                                      | CPU resource limit for the agent.                                                                                                                                                                                                                                        |
 | `ARGO_AGENT_MEMORY_LIMIT`              | `resource.Quantity` | `256m`                                                                                      | Memory resource limit for the agent.                                                                                                                                                                                                                                     |
+| `ARGO_REDACT_POD_LOGS`                 | `bool`              | `false`                                                                                      | Whether to redact pod logs to hide/mask secrets.                                                                                                                                                                                                                        |
```

Comment:
I think this should be turned on by default.

Reply:
Although I am not sure if we should turn on new features by default. WDYT? @sarabala1979 @alexec